_includes/header.html

```diff
@@ -0,0 +1,59 @@
+
+<!-- header -->
+<header class="header-cover">
+  <a href="http://dashingdemo.herokuapp.com/sample" target="_blank" class="header-side-btn header-side-btn--left">
+    <i class="fa fa-desktop fa-3x"></i><br><small>Demo</small>
+  </a>
+  <a href="https://github.com/Smashing/smashing" target="_blank" class="header-side-btn header-side-btn--right">
+    <i class="fa fa-github-alt fa-3x"></i><br><small>GitHub</small>
+  </a>
+
+  <div class="container">
+
+    <nav id="navigation" class="navbar">
+      <div class="container-fluid">
+
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+						<span class="sr-only">Toggle navigation</span>
+						<span class="icon-bar"></span>
+						<span class="icon-bar"></span>
+						<span class="icon-bar"></span>
+					</button>
+
+          <a class="navbar-brand js-smoothscroll" data-bypass="true" href="{{ site.url }}">
+            <img src="{{ '/assets/images/smashing-logo-sm-negative.png' | prepend: site.url }}" alt="Libre" height="" />
+          </a>
+        </div>
+
+        <!-- navbar links -->
+        {% include nav.html %}
+
+      </div>
+    </nav>
+
+
+    <div class="col-md-offset-2 col-md-8 header-intro">
+      <div class="header-intro--text">
+        <h1>
+          <img src="{{ '/assets/images/smashing-home-text.svg' | prepend: site.url }}" alt="Smashing" width="100%"
+                                                                   />
+          <small>Sinatra based framework that lets you<br> build excellent dashboards</small>
```

Comment:
The `line-height` of this `small` is all mucked up on my browser, the text runs together. Could be fixed if we make this `small` into a `span` and change the `h1` to and `h2`.

Reply:
Great... I will fix it!  Thanks. :)

Reply:
Changed line-height and text element. 

Screenshot using Safari Browser and I tested at Firefox too.
![image](https://user-images.githubusercontent.com/114177/41846350-cb267c10-784c-11e8-983d-1f5173780ff8.png)